libs/packages/components/src/lib/autocomplete-search/autocomplete-search.component.ts

```diff
@@ -538,6 +538,12 @@ export class SDSAutocompleteSearchComponent implements ControlValueAccessor {
       }
     }
   }
+  getClass() {
+    return this.configuration.inputReadOnly &&
+      this.configuration.selectionMode === SelectionMode.MULTIPLE
```

Comment:
Why is it hiding cursor only for multiple? Dropdown mode should work for single as well?

Reply:
for Single, the hide doesn't work because the color transparent and selected text will be inside the input.